### PR TITLE
Update recipe to 3.2.0

### PR DESCRIPTION
**Pull request by souravsingh**

### CODE REVIEW

**Comment by souravsingh** [comment]
@conda-forge-admin, please rerender

**Comment by souravsingh** [comment]
@jakirkham I used all the suggestions mentioned [here](https://github.com/conda-forge/mosfit-feedstock/issues/23#issuecomment-349016207)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -54,7 +57,10 @@ test:
     - gensim.summarization
     - gensim.test
     - gensim.topic_coherence
+  files:
+    - forge_test.sh
   commands:
+    - time bash ./forge_test.sh  # [not win]
     - nosetests --exe -v gensim
```
Comment (jakirkham) [inline recipe/meta.yaml]: This would need to move into the shell script to work.

**Comment by souravsingh** [comment]
@menshikh-iv The PR supersedes #8. Would you like to take a look and see if any changes need to be made?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -42,6 +42,8 @@ test:
     - morfessor
     - scikit-learn
     - pyemd
+    - mock
+    - boto3
```
Comment (menshikh-iv) [inline recipe/meta.yaml]: `boto3` isn't gensim dependency

File: appveyor.yml [hunk 3]
```diff
@@ -39,24 +39,24 @@ install:
 
     # Add path, activate `conda` and update conda.
     - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
-    - cmd: conda update --yes --quiet conda
+    - cmd: conda.exe update --yes --quiet conda
```
Comment (menshikh-iv) [inline appveyor.yml]: only question - what's a reason for it?

File: recipe/forge_test.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
```
Comment (menshikh-iv) [inline recipe/forge_test.sh]: Why you "hide" testing in this file?

File: recipe/meta.yaml (hunk 2 above)
Comment (souravsingh) [reply recipe/meta.yaml]: For some reason, the import tests throw an error due to boto3 dependency for smart_open. That's why I added it.

File: appveyor.yml (hunk 3 above)
Comment (souravsingh) [reply appveyor.yml]: This commit is made by the conda-forge admin. It is just for updating the testing for Windows platform and to update to latest version of Conda for testing.

File: recipe/forge_test.sh (hunk 4 above)
Comment (souravsingh) [reply recipe/forge_test.sh]: The script contains environment variables for `SSL_CERT_FILE` and `REQUESTS_CA_BUNDLE`, without unsetting them beforehand, the downloader tests fail. So I used the solution used by people [here](https://github.com/conda-forge/mosfit-feedstock/issues/23#issuecomment-349016207) for running the test suite for Linux.

**Comment by souravsingh** [comment]
Regarding Appveyor tests, The CI just runs the import tests. The latest commit should fix it.

**Comment by menshikh-iv** [comment]
Thanks @souravsingh :+1:

**Pull request merged**